Migration guide for canRequestFocus deprecation (#8994)
flutter/website: src/release/breaking-changes/can-request-focus.md

---
title: TextField.canRequestFocus Deprecated
description: >
  `TextField`'s `canRequestFocus` parameter is deprecated and replaced by the
  `canRequestFocus` parameter of its `FocusNode`.
---

## Summary

`TextField.canRequestFocus` is deprecated. The same functionality can be
achieved by setting the `canRequestFocus` parameter of the `TextField`'s
`FocusNode`.

## Background

> `TextField.canRequestFocus` was added in order to support `DropdownMenu`, which
> has a TextField that sometimes is not interactive. However, the same

Reviewer: ```suggestion
`TextField.canRequestFocus` was added to support `DropdownMenu`, which
has a `TextField` that sometimes isn't interactive. However, the same
```
